authors/apps/profiles/models.py

```diff
@@ -0,0 +1,47 @@
+from datetime import date
+
+from django.db import models
+from django.db.models.signals import post_save
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from authors.apps.authentication.models import User
+
+
+UserModel = getattr(settings, "AUTH_USER_MODEL", User)
+
+
+class Profile(models.Model):
+    """
+       Create a data model to hold more information,
+       about our awesome users.
+       It is through profiles that we get to know
+       more about the name behind a user account.
+    """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=120, blank=True, null=True)
+    last_name = models.CharField(max_length=120, blank=True, null=True)
+    date_of_birth = models.DateField(_("Date"), blank=True, null=True)
+    bio = models.TextField(blank=True, null=True)
+    image = models.URLField(blank=True,max_length=200, default="url")
+    following = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
@Rhytah , Why are you using `models.URLField()` on the image and not `models.ImageField()`.

Reply:
does models.ImageField store images into the database?

Reply:
@ManuelDominic currently as we are working on the backend, a `URLField` is more appropriate. It will store a link to the image that user chooses.

Reply:
@sebbss  yes `ImageField` does store an image in the database, but we can use a `URLField` as well to store the `URL` of the image in the database.

Reply:
@mirest/space-sprinters , @Rhytah approach is okay. Images can be stored on another server which returns image URLs so that we don't overload our database